pkgs/desktops/lxqt/lxqt-openssh-askpass/default.nix

```diff
@@ -47,5 +47,6 @@ mkDerivation rec {
     license = licenses.lgpl21Plus;
     platforms = platforms.linux;
     maintainers = teams.lxqt.members;
+    mainProgram = pname;
```

Comment:
A small nit. 😇

```suggestion
    mainProgram = "lxqt-openssh-askpass";
```

Reply:
I guess I misread the style guide. Thanks!